insight-generator/elastic_index.sh

```diff
@@ -0,0 +1,283 @@
+#!/bin/bash
+#!usr/bin/env python .
+
+source env.sh
+
+cd requirements
+
+bin/elasticsearch
+
+curl -XDELETE 'localhost:9200/insight-generator'
+
+# create ElasticSearch index
+curl -XPOST 'http://localhost:9200/insight-generator' -d '{
+  "settings": {
+    "index": {
+      "mapping.allow_type_wrapper": true
+    }
+  }
+}'
+
+# create mapping
+curl -X POST -d '{
+      "docs": {
```

Comment:
this whole JSON should be put in a file. Let's call it di-mapping-schema.json. Then the `-d` command should be updated above to -d '`cat di-mapping-schema.json`'

Reply:
will  do.